lib/opensusebasetest.pm

```diff
@@ -651,4 +651,10 @@ sub post_fail_hook {
     save_screenshot;
 }
 
+sub assert_true {
```

Comment:
I wonder if we can actually reuse what is already in `Test` and `Test::More` packages.

Reply:
Yes, good point. Just didn't know about that method, but wanted to show in the prototype what I want to achieve. 

Will try to use this.